Entities/Scene.cpp

```diff
@@ -916,9 +919,14 @@ int Scene::LoadData(bool placeObjects, bool initPathfinding, bool placeUnits)
     if (initPathfinding)
     {
         // Create the pathfinding stuff based on the current scene
-        m_pPathFinder = new PathFinder(this, 20, 2000);
-        // Update all the pathfinding data
-        m_pPathFinder->RecalculateAllCosts();
+		int sceneArea = GetWidth() * GetHeight();
+		int pathfinderGridNodeSize = g_SettingsMan.GetPathFinderGridNodeSize();
+		unsigned int numberOfBlocksToAllocate = 4000;//std::min(128000, sceneArea / (pathfinderGridNodeSize * pathfinderGridNodeSize));
```

Comment:
Did you ever try using a non-fixed number of allocated blocks, with your team-based pathfinders and less frequent micropather resets?
I know when I tried this, gotcha reported slowdowns, but with things not resetting multiple times a frame, I expect that might change.

If you don't want this, or don't wanna deal with it, I'd say move the commented code into a todo or a github issue so it's not forgotten about.
Also, note that the old default block number was 2000, though bumping it up to 4000 seemed to help even with constant resets.